### PR TITLE
Update entity components

### PR DESCRIPTION
**Pull request by joostlek**
SSIA

### CODE REVIEW

File: services/bots/src/github-webhook/github-webhook.const.ts [hunk 1]
```diff
@@ -75,12 +75,14 @@ export const entityComponents = new Set([
   'date',
   'datetime',
   'device_tracker',
+  'diagnostics',
```
Comment (frenck) [inline services/bots/src/github-webhook/github-webhook.const.ts:78]: Diagnostic is a platform, not an entity component?

**Review by frenck — CHANGES_REQUESTED** [review]

File: services/bots/src/github-webhook/github-webhook.const.ts (hunk 1 above)
Comment (joostlek) [reply services/bots/src/github-webhook/github-webhook.const.ts:78]: Oh true

Comment (frenck) [reply services/bots/src/github-webhook/github-webhook.const.ts:78]:
```suggestion
```

**Review by frenck — APPROVED** [review]
Thanks, @joostlek 👍 

../Frenck

**Pull request merged**